lm_engine/utils/packages.py

```
@@ -53,6 +53,18 @@ def is_colorlog_available() -> bool:
     return _IS_COLORLOG_AVAILABLE
 
 
+try:
+    import cutlass.cute
+
+    _IS_CUTE_DSL_AVAILABLE = True
+except:
```

Ruff (0.16.3)

[error] Do not use bare `except`

(E722)